.github/workflows/ci.yml

```
@@ -62,7 +62,7 @@ jobs:
           name: code-coverage-ubuntu-latest-20.x
           path: coverage/
       - name: SonarCloud Scan
-        uses: SonarSource/sonarcloud-github-action@v2.2.0
+        uses: SonarSource/sonarcloud-github-action@v2.3.0
```

```suggestion
        uses: SonarSource/sonarcloud-github-action@v2
```

It would be better to just use the major version tag for GH actions